### ISSUE TITLE
remove help() from startup banner

### ISSUE THREAD

**Issue by andrewcooke** [JuliaLang/julia#12662]
i have no idea where the issue is for related work - i've tried searching on help but not found anything.

anyway, apparently you've decided to remove help, but it's still there in the startup banner.  which is a little confusing.

**Comment by joehuchette** [comment]
55339fca71e643db614c459c55fd742fdef8c4d0 (@one-more-minute)

from [julia users](https://groups.google.com/forum/#!topic/julia-users/3iUo5Q7s8FI)

**Comment by one-more-minute** [comment]
Could we just replace `help()` with `??`?

**Comment by joehuchette** [comment]
Also a little funny:
```
help?> ?
search:

  Welcome to Julia 0.4.0-dev+6760. The full manual is available at

  http://docs.julialang.org/

  as well many great tutorials and learning resources:

  http://julialang.org/learning/

  For help on a specific function or macro, type ? followed by its name, e.g. ?fft, ?@time or
  ?html, and press enter.

help?> html
search: HTML @html_str gethostname

Couldn't find html
Perhaps you meant htol, hton, hcat, hist, hash, hex, chol, chomp, ltoh, ntoh, call, eval or @cmd
```

**Comment by one-more-minute** [comment]
Looks like that was meant to be `html""`. Let's just replace that with `r""`.

**Comment by JeffBezanson** [comment]
fixed by #12707
